src/Libraries/DSOffice/Excel.cs

```diff
@@ -350,6 +350,14 @@ private static object[][] ConvertToJaggedArray(object[,] input)
                     cols = Math.Max(cols, input[i].GetUpperBound(0) + 1);
             }
 
+            // if the input data is an empty list or a list of nested empty lists
+            // return an empty cell
+            if(rows == 0 || cols == 0)
```

Comment:
Shall we change the condition to rows == 0 && cols ==0?

If rows != 0, and cols ==0, shall we return { "", "", "", ...}? E.g., if rows is 2, then return {"", ""}?